### PR TITLE
Vacation requests list

### PR DESCRIPTION
**Pull request by bigardone**
Vacation requests list implemented. Pending vacation requests are shown and they can be approved or rejected from the list itself.

captura de pantalla 2015-04-27 a las 15 56 52

### CODE REVIEW

File: app/controllers/api/v1/vacation_requests_controller.rb [hunk 1]
```diff
@@ -9,14 +15,55 @@ def create
...
+
+    vacation_request = VacationRequest.find params[:id]
+    vacation_request.reject!
+
+    render json: vacation_request, status: :ok
+  end
+
   private
 
   def vacation_request_params
```
Comment (bbay) [inline app/controllers/api/v1/vacation_requests_controller.rb:50]: nice! I didnt think of this way of separating the params for different users :dancer:

Comment (bigardone) [inline app/controllers/api/v1/vacation_requests_controller.rb:50]: Strong params FTW :)

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -11,11 +11,12 @@ class ApplicationController < ActionController::Base
...
     render head: :forbidden, text: 'Forbidden' unless current_user && current_user.admin?
   end
-  
+
   private
 
   def set_gon_globals
     gon.authenticity_token = form_authenticity_token
     gon.locale = I18n.locale
+    gon.vacation_request_statuses = VacationRequest.statuses
```
Comment (javiercr) [inline app/controllers/application_controller.rb:20]: I wasn't sure the plural of _status_ was actually _statuses_ but it seems it is lol

**Comment by javiercr** [comment]
:+1: looking great!

**Pull request merged**